packages/core/src/components/external/PlanningView/PlanningView.tsx

```diff
@@ -299,13 +333,21 @@ const getStyles = Colors =>
     circleButton: {
       marginHorizontal: 5,
       marginTop: 5,
-      width: 50 + 50 * (1 / 3),
+      width: 40 + 50 * (1 / 3),
     },
     headerPlanning: {
+      flexDirection: assignedToMeSwitch ? 'row' : 'row-reverse',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    headerButton: {
       flexDirection: 'row',
       alignItems: 'center',
-      alignContent: 'flex-end',
-      alignSelf: 'flex-end',
+      marginRight: 10,
+    },
+    switchCard: {
+      width: '45%',
+      marginHorizontal: 10,
     },
```

Comment:
On my screen, calendar button are close to right border of the screen, the margin is not used because the line is too big for the screen. So reduce all margin and/or reduce switch and buttons size